docs/source/developer-guidelines/contributing.md

```diff
@@ -9,8 +9,16 @@ myst:
 
 # Contributing to Volto
 
-You may have an issue to report, make a feature request, report a security vulnerability, or you want to create a pull request.
-You have come to the right place to learn how to do so.
+First read {doc}`plone:contributing/index`.
+Volto follows those guidelines with a few specific variations, as described in this chapter.
+
+If after reading this you become hesitant, don't worry.
+You can always create a pull request, mark it as "Draft", and improve the following points later, requesting help from the community.
+
+
+## Sign and return the Plone Contributor Agreement
+
+See {ref}`Plone Contributor Agreement <contributing-sign-and-return-the-plone-contributor-agreement-label>`.
 
 
 ## Reporting an issue or making a feature request
```

Comment:
@stevepiercy Should we put this section above the one about the contributor agreement? the CA is required for making a pull request, but not for submitting issues.